session/bootstrap.go

```diff
@@ -1335,6 +1335,13 @@ func doDMLWorks(s Session) {
 					vVal = string(variable.DynamicOnly)
 				}
 			}
+			if v.Name == variable.TiDBEnableMultiSchemaChange {
+				vVal = variable.BoolOff
+				if flag.Lookup("test.v") != nil || flag.Lookup("check.v") != nil {
```

Comment:
Does it mean the test will fail if we don't specify "-v" option?

Reply:
Nope, if you don't specify it, you will get a `false` value when the test is running, it's still not equal to nil.

Here judge whether args scope contains `test.v` or `check.v`, for eliminated value, the args position will be set to false automatically when test start. Only if you are exec `go run xxx`, there won't be any scope for `test.v` or `check.v`.